sparkplug/timereporters/ddstatsd.py

```diff
@@ -0,0 +1,69 @@
+"""
+Sends timing information to Datadog custom metrics
+Uses a thread to send at regular intervals, without
+interfering with the main consumer thread
+"""
+
+from sparkplug.logutils import LazyLogger
+_log = LazyLogger(__name__)
+
+from sparkplug.timereporters.base import Base, mn_md_mx
+
+try:
+    # pip install datadog
+    from datadog import initialize
+
+    from datadog.threadstats.base import ThreadStats
```

Comment:
Would you consider putting the imports at the top of the file to improve readability of modules used?

Reply:
Hey Tobi, I'm failing safe when datadog is not available.  Catching the ImportError and defining a noop class instead of the real one.